app/models/container_limit.rb

```diff
@@ -0,0 +1,12 @@
+class ContainerLimit < ActiveRecord::Base
+  include CustomAttributeMixin
+  include ReportableMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :container_project
+
+  has_many :container_limit_items, :dependent => :destroy
+
+  has_many :labels, -> { where(:section => "labels") }, :class_name => "CustomAttribute", :as => :resource
+  has_many :selector_parts, -> { where(:section => "selectors") }, :class_name => "CustomAttribute", :as => :resource
```

Comment:
@moolitayer actually I found the `labels` (although I am not sure if we want to collect them here) but I can't find the `selector` yet.